[2025-06-28T10:22:37.639+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_classic manual__2025-06-28T10:22:07.401451+00:00 [queued]>
[2025-06-28T10:22:37.643+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_classic manual__2025-06-28T10:22:07.401451+00:00 [queued]>
[2025-06-28T10:22:37.643+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-06-28T10:22:37.649+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonVirtualenvOperator): virtualenv_classic> on 2025-06-28 10:22:07.401451+00:00
[2025-06-28T10:22:37.660+0000] {standard_task_runner.py:57} INFO - Started process 181 to run task
[2025-06-28T10:22:37.664+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator', 'virtualenv_classic', 'manual__2025-06-28T10:22:07.401451+00:00', '--job-id', '15', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_python_operator.py', '--cfg-path', '/tmp/tmp2cb5hqru']
[2025-06-28T10:22:37.669+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask virtualenv_classic
[2025-06-28T10:22:37.727+0000] {task_command.py:416} INFO - Running <TaskInstance: example_python_operator.virtualenv_classic manual__2025-06-28T10:22:07.401451+00:00 [running]> on host 4eaa1e3302ef
[2025-06-28T10:22:37.775+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator' AIRFLOW_CTX_TASK_ID='virtualenv_classic' AIRFLOW_CTX_EXECUTION_DATE='2025-06-28T10:22:07.401451+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-28T10:22:07.401451+00:00'
[2025-06-28T10:22:37.781+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv9wttb7ph --system-site-packages
[2025-06-28T10:22:37.790+0000] {process_utils.py:186} INFO - Output:
[2025-06-28T10:22:38.778+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 177ms
[2025-06-28T10:22:38.788+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venv9wttb7ph, clear=False, no_vcs_ignore=False, global=True)
[2025-06-28T10:22:38.788+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-06-28T10:22:38.789+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.3.1, setuptools==68.2.2, wheel==0.41.2
[2025-06-28T10:22:38.789+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-06-28T10:22:38.809+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv9wttb7ph/bin/pip install -r /tmp/venv9wttb7ph/requirements.txt
[2025-06-28T10:22:38.819+0000] {process_utils.py:186} INFO - Output:
[2025-06-28T10:22:39.959+0000] {process_utils.py:190} INFO - Collecting colorama==0.4.0 (from -r /tmp/venv9wttb7ph/requirements.txt (line 1))
[2025-06-28T10:22:39.960+0000] {process_utils.py:190} INFO -   Using cached colorama-0.4.0-py2.py3-none-any.whl.metadata (13 kB)
[2025-06-28T10:22:39.962+0000] {process_utils.py:190} INFO - Using cached colorama-0.4.0-py2.py3-none-any.whl (21 kB)
[2025-06-28T10:22:42.474+0000] {process_utils.py:190} INFO - Installing collected packages: colorama
[2025-06-28T10:22:42.477+0000] {process_utils.py:190} INFO -   Attempting uninstall: colorama
[2025-06-28T10:22:42.477+0000] {process_utils.py:190} INFO -     Found existing installation: colorama 0.4.6
[2025-06-28T10:22:42.477+0000] {process_utils.py:190} INFO -     Not uninstalling colorama at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venv9wttb7ph
[2025-06-28T10:22:42.478+0000] {process_utils.py:190} INFO -     Can't uninstall 'colorama'. No files were found to uninstall.
[2025-06-28T10:22:42.493+0000] {process_utils.py:190} INFO - Successfully installed colorama-0.4.0
[2025-06-28T10:22:42.592+0000] {process_utils.py:190} INFO - 
[2025-06-28T10:22:42.592+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.3.1 -> 25.0.1
[2025-06-28T10:22:42.592+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-06-28T10:22:42.716+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv9wttb7ph/bin/python /tmp/venv9wttb7ph/script.py /tmp/venv9wttb7ph/script.in /tmp/venv9wttb7ph/script.out /tmp/venv9wttb7ph/string_args.txt /tmp/venv9wttb7ph/termination.log
[2025-06-28T10:22:42.734+0000] {process_utils.py:186} INFO - Output:
[2025-06-28T10:22:43.820+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-28T10:22:43.851+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=virtualenv_classic, execution_date=20250628T102207, start_date=20250628T102237, end_date=20250628T102243
[2025-06-28T10:22:43.897+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-28T10:22:43.912+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
